packages/contracts/package.json

```diff
@@ -8,15 +8,15 @@
     "chain": "yarn rm-dbdata && anvil --config-out localhost.json",
     "clean": "forge clean && rm -rf broadcast cache deployments types localhost.json index.js",
     "dev": "yarn rm-dbdata && anvil --config-out localhost.json & make deploy-local",
-    "build": "forge build && yarn gen-types",
+    "build": "forge build && yarn gen-types && tsc",
     "test": "forge test",
     "format": "forge fmt && prettier './script/**/*.js' --write",
     "lint": "forge fmt && eslint --fix --ignore-path .gitignore && yarn solhint './*(test|src)/**/*.sol'",
     "db-check": "tsc && node ./script/utils/checkDb.js",
     "db-export": "tsc && node ./script/utils/exporter.js",
     "deploy": "make deploy-local",
-    "deploy:ci": "yarn db-export && make deploy-baseSepolia",
```

Comment:
this is a bit odd.  before we were calling db-export -> deploy -> db-export?

Reply:
Yes, it was a bit odd :)